_posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md

```diff
@@ -0,0 +1,87 @@
+
+One of the fabulous things about this project being open source is that there has been global adoption, especially in areas of the world where it would be difficult to obtain a license to proprietary software.  The individuals who have taken to this project are from many languages like Python, Julia, and R.  Subjects that seem to utilize it the most range from science to digital humanities and some even use it for operations and training.  The contributors tend to be those who were originally just delighted users that wanted to continue building on this project that they enjoyed using.  There are many people who can see the value in this project and want to give back to it.  Educators have been instrumental in its continued growth at the highest levels as well as community workshops etc.  The enterprise crowd has even been utilizing it for things like machine learning.  Far a showcase of these uses you can browse the [Jupyter Blog](https://blog.jupyter.org/).
+
+To help encourage diversity, they will often take the time to message new community members on an individual basis and ensure that they can feel comfortable participating in the community.  In addition to this, some of the Jupyter funding is reserved for diversity and inclusion efforts and so far there are two people who are working on [this program](https://blog.jupyter.org/outreachy-jupyter-supporting-diversity-in-open-communities-dfa78db4b0bd) by reaching out to people in an effort to bring them into the community.  Inclusion is a huge topic, so to make efforts more effective Carol tries to look at things based on their “multiplier Effect”.  One of the big things that was done in the Python community was to create a welcoming and thoughtful community.  The bar was constantly raised to ensure that the expectation for kindness and graciousness was always higher.  Simple things like using emojis helped to lighten the mood and achieve these goals of being more inviting.  Documentation is another critical aspect which allows people in the Jupyter Ecosystem to “learn how to fish”.  While it is desirable to do more in this area, many organizations are limited in their resource allocation to diversity efforts.  To address this it is important to help them understand that many of these efforts are just generally designed to grow the community and as that happens diversity will come.  
+
+
+
+### Demo
+
+[Demo at timestamp 34:13](https://www.youtube.com/watch?v=HzyfFSLfHiQ)
```

Comment:
This link should be to the timestamped location, ie https://youtu.be/HzyfFSLfHiQ?t=2053